modules/ocf_kubernetes/manifests/master/loadbalancer.pp

```diff
@@ -61,6 +61,15 @@
     + [{'redirect'      => 'scheme https code 301 if !{ ssl_fc }'}],
     require => Class['ocf_kubernetes::master::loadbalancer::ssl'];
   } ->
+  haproxy::frontend { 'kubernetes-proxy-frontend':
+    # This is used for hosts that don't directly point to lb-kubernetes, but
+    # are instead reverse proxied from another server (like puppet, www, irc)
```

Comment:
You should add one more line about the logic behind this new frontend that points to the same backend -- that is our proxy server already handles ssl termination, and using the `kubernetes-frontend` will force ssl.

Reply:
done, updated the comment